shortenedurl/impl/src/java/org/sakaiproject/shortenedurl/impl/BitlyUrlService.java

```diff
@@ -179,4 +179,12 @@ public void setApiKey(String apiKey) {
 		this.apiKey = apiKey;
 	}
 
+	/*
+	 * this is a placeholder until someone who understands bitly decides what the
+	 * right approach is
+	 */
+	public boolean shouldCopy(String url) {
+		return false;
+	}
+
```

Comment:
IMO its fine for URLs to no longer resolve in this particular case, implementors can handle it with an appropriate error page. Though a note should be added to the default.sakai.properties about this particular issue.